Documentation/troubleshooting.rst

```diff
@@ -31,15 +31,21 @@ fails between endpoints across multiple nods.
 Troubleshooting steps:
 ~~~~~~~~~~~~~~~~~~~~~~
 
-1. Run `cilium monitor` on the node of the source and destination endpoint.
+1. Run `cilium-health status` on the node of the source and destination
```

Comment:
Unrelated to this PR, could you also fix the "multiple nods" typo on line 29? (amusing as it is!)